src/client/ui/controls.js

```diff
@@ -180,7 +182,7 @@ export default class StorybookControls extends React.Component {
             onClear={this.clearFilterText.bind(this)}
           />
         </div>
-        <div style={listStyle}>
+        <div style={listStyle} className="stories-list">
```

Comment:
Do not use classNames just for the sake of testing.
May be we can simply get the html `.html()` with enzyme and do some regexp search for what you want to search.